### PR TITLE
Add bundlesize

### PR DESCRIPTION
**Pull request by timneutkens**

### CODE REVIEW

**Comment by timneutkens** [comment]
DM'ed @siddharthkp on twitter to see why it's throwing an error, the token seems to have the correct rights.

**Comment by siddharthkp** [comment]
Hi :wave:

If you run it with the `--debug` flag, it tries to tell you if the required keys are present

**Comment by siddharthkp** [comment]
```
token exists: true 
repo: "zeit/next.js" 
api enabled: true 
```

This is good! My guess is that the GitHub token does not permission to set a status. (GitHub gives a 404 for unauthenticated as well)

File: package.json [hunk 1]
```diff
@@ -32,6 +33,38 @@
...
+      "maxSize": "1.46 kB"
+    },
+    {
+      "path": "./test/integration/size-limit/.next/static/*/pages/about.js",
+      "maxSize": "237 B"
+    },
+    {
+      "path": "./test/integration/size-limit/.next/static/chunks/*.js",
+    "jest-junit": "^5.0.0",
+    "jest-junit": "^5.0.0",
```
Comment (tusbar) [inline package.json]: These lines probably don’t help 😄

Comment (timneutkens) [reply package.json]: agreed, my git gui screwed it up I think 🤔

**Comment by timneutkens** [comment]
Since bundlesize doesn't expose the size without providing a github token we can't use it for public PRs which I intended it for. So I'll close this PR.

**Comment by siddharthkp** [comment]
> Since bundlesize doesn't expose the size without providing a github token we can't use it for public PRs which I intended it for. So I'll close this PR.

bundlesize uses github token to set the GitHub status. You can still use it on the CI but it wouldn't be able to push the results back on the PR

That's a huge limitation that I don't know how to bypass 🤷‍♀️(are GitHub apps the answer, idk?)

**Pull request closed without merging**